src/ui/RealmBrowser/focus.ts

```diff
@@ -43,7 +44,7 @@ interface IFocus {
 
 export interface IClassFocus extends IFocus {
   kind: 'class';
-  results: Realm.Results<any>;
+  results: Realm.Results<any> | AsymmetricObjectCollection<any>;
```

Comment:
Don't think the union should be needed as it implements `Realm.Results`.
```suggestion
  results: Realm.Results<any>;
```